storey/flow.py

```diff
@@ -629,3 +664,150 @@ def build_flow(steps):
+            response = await client_session.put(f'{self._webapi_url}/{self.table}/{key}',
+                                                headers=self._put_item_headers, data=json.dumps(data), ssl=False)
+            if not response.status == 200:
+                body = await response.text()
+                raise V3ioError(f'Failed to save aggregation for key: {key}. Response status code was {response.status}: {body}')
+
+        await client_session.close()
+
+    async def save_key(self, key, aggr_item):
+        connector = aiohttp.TCPConnector()
+        client_session = aiohttp.ClientSession(connector=connector)
+
+        data = {'Item': self._get_attributes_as_blob(aggr_item), 'UpdateMode': 'CreateOrReplaceAttributes'}
+        response = await client_session.put(f'{self._webapi_url}/{self.table}/{key}',
+                                            headers=self._put_item_headers, data=json.dumps(data), ssl=False)
+        if not response.status == 200:
+            body = await response.text()
+            raise V3ioError(f'Failed to save aggregation for key: {key}. Response status code was {response.status}: {body}')
+
+        await client_session.close()
+
+    def _get_attributes_as_blob(self, aggregation_element):
+        data = {}
+        for name, bucket in aggregation_element.aggregation_buckets.items():
+            # Only save raw aggregates, not virtual
+            if bucket.should_persist:
+                blob = pickle.dumps(bucket.to_dict())
+                data[name] = {'B': base64.b64encode(blob).decode('ascii')}
+
+        return data
+
+    async def load_store(self):
+        pass
+
+    # Loads a specific key from the store, and returns it in the following format
+    # {
+    #   'feature_name_1': {'first_bucket_time': <time> 'values': []},
+    #   'feature_name_2': {'first_bucket_time': <time> 'values': []}
+    # }
+    async def load_key(self, key):
+        connector = aiohttp.TCPConnector()
+        client_session = aiohttp.ClientSession(connector=connector)
+
+        request_body = json.dumps({'AttributesToGet': '*'})
+
+        try:
+            response = await client_session.put(f'{self._webapi_url}/{self.table}/{key}',
+                                                headers=self._get_item_headers, data=request_body, ssl=False)
+            body = await response.text()
+            if response.status == 404:
+                return None
+            elif response.status == 200:
+                parsed_response = _v3io_parse_get_item_response(body)
+                for name, blob in parsed_response.items():
+                    parsed_response[name] = pickle.loads(blob)
+
+                return parsed_response
+            else:
+                raise V3ioError(f'Failed to get item. Response status code was {response.status}: {body}')
+        except BaseException as ex:
+            raise ex
```

Comment:
I don't think this except clause does anything... It just catches to raise the exception.